value/src/main/java/com/google/auto/value/processor/autovalue.vm

```diff
@@ -201,7 +204,7 @@ final class $subclass$formalTypes extends $origClass$actualTypes {
   #if ($builderIsInterface) implements #else extends #end
       ${builderTypeName}${builderActualTypes} {
 
-    private final $bitSet set$ = new ${bitSet}(${props.size()});
+    private final $bitSet set$ = new ${bitSet}();
```

Comment:
The non-nullable property count is not readily available, and the default BitSet size is 64 bits, so it would only make a difference if there were more than 64 properties anyway.